src/webhooks/interfaces/webhook.interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface Webhook {
+  id: string;
+  event: string;
+  data: object;
```

Comment:
We should use `Record<string, any>` here to model an untyped object, as using `object` does not allow access to arbitrary keys by the SDK consumer.

```suggestion
  data: Record<string, any>;
```

Eventually we'll want to have typed representations of our webhook payloads.